larray/core/array.py

```diff
@@ -643,6 +642,17 @@ def nan_equal(a1, a2):
     return element_equal(a1, a2, nan_equals=True)
 
 
+def _handle_deprecated_argument_title(meta, title):
+    if meta is None:
+        meta = Metadata()
+    if title is not None:
+        import warnings
+        warnings.warn("title argument is deprecated. Please use meta argument instead", FutureWarning, stacklevel=2)
+        meta['title'] = title
+        title = None
+    return meta, title
```

Comment:
AFAICT, you don't have to return title since it is (or at least should) never (be) used